shapely/geometry/base.py

```diff
@@ -490,6 +513,8 @@ def buffer(
             CAP_FLAT.
         quadsegs : int, optional
             Deprecated alias for `quad_segs`.
+        **kwargs : dict, optional
+            For backwards compatibility, will be ignored.
```

Comment:
Those will not actually be ignored, but we raise an error if there are unexpected keyword arguments